src/v0/destinations/google_adwords_offline_conversions/config.js

```diff
@@ -42,6 +42,8 @@ const CONVERSION_CUSTOM_VARIABLE_CACHE_TTL = process.env.CONVERSION_CUSTOM_VARIA
 
 const MAPPING_CONFIG = getMappingConfig(CONFIG_CATEGORIES, __dirname);
 
+const consentFields = ['adUserData', 'adPersonalization'];
```

Comment:
Can we move it to the common GoogleUtils?

Reply:
you can rename them to adConsentFields

Reply:
moved to common utils

Reply:
Each Google destination may have specific requirements for consent fields. Moving it to common utils is not a good idea.

Reply:
That's what has been suggested - `you can rename them to adConsentFields in GoogleUtils`

And this is constant defined now - `const **ADWORDS**_CONSENT_FILEDS = ['adUserData', 'adPersonalization'];` Anything else you are suggesting ?

Reply:
I'm good with the renaming, but we should be able to use this constant without hard-coding in the utility function itself.

Instead, let the destination handlers import this constant and pass it back to the utility function as an input.

This way, we're not duplicating the consent field definitions across all the "ads" destinations and the utility implementation is still generic to handle any consent fields (analytics consent fields also in the future).

Reply:
We should avoid this,
https://github.com/rudderlabs/rudder-transformer/blob/9bae3d455cf254d0b315ce588bc7ba7bc66e1424/src/v0/util/googleUtils/index.js#L74

Reply:
@krishna2020 @saikumarrs please take a look. Updated the util function